web/src/elements/sidebar/SidebarItem.ts

```diff
@@ -117,19 +117,11 @@ export class SidebarItem extends AKElement {
         if (!this.path) {
             return false;
         }
-        if (this.path) {
```

Comment:
I realized the two comparisons were complimentary, so that takes out the `if`.  `exec` isn't needed here, just `test`.  Giving the two comparisons names helps understand what the code is doing.

// prefer-single-boolean-return